### DISCUSSION TITLE
BluetoothClassic support

### DISCUSSION THREAD

**Discussion by shaboigan** [ESPresense/ESPresense#320] [General]
I've come from room-assistant that tracked ble and classic, I saw a port to esp request that lead me over here, but I'm not seeing explicit support for Bluetooth classic

Am I missing something in the config or does espresense only handle ble currently?

**Comment by DTTerastar** [comment]
It's only BLE and can't really be expected to do both at the same time.  It's just a little microcontroller, not a full linux running chip...
